app/jobs/restart_pipeline_for_sample.rb

```diff
@@ -0,0 +1,20 @@
+class RestartPipelineForSample
+  @queue = :q03_pipeline_run
+
+  def self.perform(sample_id)
+    s = Sample.find(sample_id)
+    if s.nil?
+      raise Exception("Sample #{sample_id} not found")
+    end
+    Rails.logger.info("RestartPipelineForSample #{sample_id} is being triggered")
+    unless s.kickoff_pipeline
+      raise Exception("not restarted")
```

Comment:
nit: I would just use the same lingo as the call, something like "Kickoff pipeline not successful" ? It is just easy to relate to the code...

Reply:
done